dashboards/node-overview.json

```diff
@@ -85,6 +85,18 @@
         }
       }
     },
+    {
```

Comment:
Steve - couple of minor requests:
(1) could you change the title a little more descriptive of what this panel will show. E.g. "Host CPU utilization"
(1) would you mind adding a description field to this metric that explains how this is different from `sys_cpu_utilization_rate`. E.g. Metric tracks host-level CPU; `sys_cpu_utilization_rate` will be container-level if the server is running in a container.